plugins/sidebar/file-list.js

```diff
@@ -4,6 +4,36 @@ const React = require('react');
 const List = require('react-material/components/List');
 
 const FileList = React.createClass({
+
+  componentDidMount: function(){
+    app.keypress.register('CTRL_TAB', this.previousFile);
+    app.keypress.register('CTRL_SHIFT_TAB', this.nextFile);
+  },
+  componentWillUnmount: function(){
+    app.keypress.register('CTRL_TAB');
+    app.keypress.register('CTRL_SHIFT_TAB');
+  },
+  previousFile: function(){
+    this.changeFile(true);
```

Comment:
boolean arguments are bad, this should be an object with `direction` property.  e.g. `{ direction: 'next' }` `{ direction: 'prev' }` or something similar